logic/src/commonMain/kotlin/com/wire/kalium/logic/data/conversation/ConversationGroupRepository.kt

```diff
@@ -68,7 +69,10 @@ internal class ConversationGroupRepositoryImpl(
             .flatMap { conversationResponse ->
                 val teamId = selfUser.teamId
                 val conversationEntity = conversationMapper.fromApiModelToDaoModel(
-                    conversationResponse, mlsGroupState = ConversationEntity.GroupState.PENDING_CREATION, teamId
+                    conversationResponse,
+                    mlsGroupState = ConversationEntity.GroupState.PENDING_CREATION,
+                    teamId,
+                    DateTimeUtil.currentIsoDateTimeString()
```

Comment:
Same here, we should use date-time from the server, here for conversations maybe we can use `lastEventTime`?